client/src/lib/components/sandbox/play/Arena.svelte

```diff
@@ -110,6 +258,18 @@
 			{#if selectedPiece}
 				<div></div>
 			{/if}
+			{#if orders}
+				<div></div>
+			{/if}
```

Comment:
Idea for a separate component:

`<ArenaToolTip>` which could also be a wrapper if different types of tooltips will be wanted, or can just be another example of shuttling some code out to a fresh class.